### ISSUE TITLE
Ratio variables attributed non-xclim units

### ISSUE THREAD

**Issue by aulemahal** [Ouranosinc/pymerra2#10]
In `download.py`, in `subdaily_netcdf()`, line 517, it does:

```python
 # 3.1. Units
# Force kg kg-1 to 1
if var_ref.units == "kg kg-1":
    var1.units = "1"
else:
    var1.units = var_ref.units
```
But, `"1"` is not something xclim understands as a unit. So any attempt at converting fails. Example, with the specific humidity at the surface "huss":
```python
convert_units_to(huss, "percent")
```
Raises: `AttributeError: 'int' object has no attribute 'units'`

So, 

1. What's wrong with "kg kg-1"?
2. Why is this done only in `subdaily_netcdf` and not `daily_netcdf`?

**Comment by Zeitsperre** [comment]
This was added by the previous owner of the library. If it's causing errors, feel free to change this in a PR.

**Comment by Zeitsperre** [comment]
@aulemahal good to close?

**Comment by aulemahal** [comment]
Yep. Fixed by #11.
